tests/test_core.py

```diff
@@ -0,0 +1,47 @@
+from stac_ipyleaflet.core import StacIpyleaflet
+
+
+# UI TEST CASES
```

Comment:
Wondering if we should create different files for the diff test cases - considering these should grow substantially. We can talk about different approaches though, if you'd like. IE - testing successful creation of UI components, vs testing that map layers exist when there is a sample data layers file (i.e. `BiomassLayers.csv` file to load)

Reply:
hah. you got a great point... so pytest conventions are that the test files should be prefixed with `test_{fileName_you_are_testing_after}` (article that talks about pytest conventions [here](https://www.skill2lead.com/pytest/pytest-naming-conventions.php#:~:text=File%20name%20should%20start%20with,and%20filename%20for%20visualization%20purpose.))... This is why I've been trying to break out code as much as possible in a modular / unit like way. That includes breaking it out into different files... Of course I didn't tackle all that in this pr though... because that would be a huge task that would just need to be done along the way now. But as we go, when we develop new features, lets think about building more modularly. And if there are parts in the code that could be broken out more, we can do it as we go.